src/Sub.re

```diff
@@ -5,9 +5,8 @@ module type Config = {
   // State that is carried by the subscription while it is active
   type state;
 
-  let subscriptionName: string;
-
-  let getUniqueId: params => string;
+  let name: string;
+  let id: params => string;
```

Comment:
These were overly verbose. The "subscription" qualifier is unnecessary because it's obvious from the context. An "id" is unique by definition, and `get` is an imperative smell that is unnecessary in general.

A breaking change, obviously, but seems okay since this is brand new.